pkg/rpc/convert.go

```diff
@@ -141,6 +141,7 @@ func ConvertFromRpcResults(rpcResults []*scanner.Result) []report.Result {
 		results = append(results, report.Result{
 			Target:          result.Target,
 			Vulnerabilities: vulns,
+			Type:            resultType,
```

Comment:
Can't we do like the following?
```suggestion
			Type:            result.Type,
```

Reply:
Fixed.